### ISSUE TITLE
Training phrase.

### ISSUE THREAD

**Issue by zhangyaqin032266** [bertinetto/siamese-fc#70]
Hi! Thank you for your contribution.
When I trained the network, I have a problem that "Error using vl_nnbnorm
operator(): operator():

Error in dagnn.BatchNorm/backward (line 29)
        vl_nnbnorm(inputs{1}, params{1}, params{2}, derOutputs{1}, ...

Error in dagnn.Layer/backwardAdvanced (line 120)
      [derInputs, derParams] = obj.backward ...

Error in dagnn.DagNN/eval (line 117)
  obj.layers(l).block.backwardAdvanced(obj.layers(l)) ;

Error in cnn_train_dag>processEpoch (line 253)
      net.eval(inputs, params.derOutputs, 'holdOn', s < params.numSubBatches) ;

Error in cnn_train_dag (line 105)
    [net, state] = processEpoch(net, state, params, 'train') ;

Error in experiment (line 102)
    [net, stats] = cnn_train_dag(net, imdb, batch_fn, opts.train);

Error in run_experiment (line 17)
	experiment(imdb_video, opts);
How to solve it? Thank you very much!
